pkg/api/grpcServer.go

```diff
@@ -61,9 +61,119 @@ type (
 )
 
 var (
```

Comment:
https://docs.google.com/document/d/1NcGgEzSEt0A0_qqJfA3x6ETNyomyMrhm1sBlpdorbHY/edit#heading=h.la7bf3j6tcb5 explains the update procedures for the dictionary. The reason we want to build this into the code is that it's not really config. I don't want operators to mess around with this stuff willy nilly.